cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -43,8 +43,17 @@ func resourceOCCMGCP() *schema.Resource {
 			},
 			"service_account_path": {
 				Type:     schema.TypeString,
-				Required: true,
+				Required: false,
 				ForceNew: true,
+				ConflictsWith: []string{"service_account_key"},
+				Default: "",
+			},
+			"service_account_key": {
+				Type:     schema.TypeString,
+				Required: false,
```

Comment:
It should be "Optional: true". One of optional, required, or computed must be set true.